WordPress/gutenberg
# Components: Add VisuallyHidden component

## Description

Adds a new component for rendering ScreenReaderText and the necessary CSS.
Per @youknowriad <a href="https://github.com/WordPress/gutenberg/pull/17997#issuecomment-543584382">comment here</a>


In testing this, I noticed one of the issues also for Storybook is they weren't setup to load the style.scss along with the components. This PR also adds the necessary CSS loaders in the Storybook webpack.config.js so a story can do this to include the CSS.

```
import ScreenReaderText from '../';
import '../style.scss';

export default { title: 'ScreenReaderText', component: ScreenReaderText };

export const _default = () => (
	<>
		<ScreenReaderText>
			This should not show.
		</ScreenReaderText>
		<div>
			This text will always show.
		</div>
	</>
);
```
This is my first new component being added, so not sure all the pieces that need to be properly updated.

## How has this been tested?

- Add `<ScreenReaderText> Test text. </ScreenReaderText>` and confirm it includes a div and proper style rules.

- Run Storybook: `npm run designs-system:dev` confirm ScreenReaderText does/doesn't show

## Types of changes
- Adds new component `ScreenReaderText`
- Updates Storybook to support loading .scss

## Checklist:
